src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/Config/AddAzureRmVmssExtensionCommand.cs

```diff
@@ -81,12 +81,6 @@ public class AddAzureRmVmssExtensionCommand : Microsoft.Azure.Commands.ResourceM
             ValueFromPipelineByPropertyName = true)]
         public Object ProtectedSetting { get; set; }
 
-        [Parameter(
-            Mandatory = false,
-            Position = 8,
-            ValueFromPipelineByPropertyName = true)]
-        public string Id { get; set; }
```

Comment:
The reason of removal of this parameter is this parameter (ID) is read-only in VMSS.  This was a bug in Swagger spec and this was fixed.